pkg/client/imagesignatures.go

```diff
@@ -0,0 +1,41 @@
+package client
```

Comment:
@deads2k since @mfojtik's client generation is in aren't we switching slowly to use that one instead of introducing new ones? Especially that this is a new type.